ffi/README.md

```diff
@@ -1,80 +1,39 @@
-# Experiments with Dart FFI
+# Dart FFI Samples
 
 A series of simple examples demonstrating how to call C libraries from Dart.
 
-Please note that the Dart FFI API is in active development and likely to change
-before release. This code is designed to work with *Dart version 2.5.0*.
+This code is designed to work with *Dart version 2.6.0* and above.
 
-## Instructions
+## Building native libraries
 
-The C make files are (currently) written to work on a Mac. To compile the
-libraries, go to the `c/` folder in each subproject and run:
+Each sample uses [CMake][cmake] to generate a Makefile. To build the native
+library for each sample:
 
-``` shell
-make dylib
+```bash
```

Comment:
Cool, the less instructions the better!

Does this suffice for all supported OSes? Including the visibility rules etc?

Reply:
I think cmake can configure things like visibility rules, but I haven't tried it. cc/ @domesticmouse

Reply:
First up, I’m a CMake noob, I just know it is widely used by developers doing cross platform c/c++ work. Secondly, what do you mean by visibility rules?